src/scripts/cmake/Sanitizers.cmake

```diff
@@ -0,0 +1,33 @@
+# Module to set flags for sanitizers
+
+# Inspired on http://www.stablecoder.ca/2018/02/01/analyzer-build-types.html
+
+if(CMAKE_BUILD_TYPE STREQUAL "Sanitizer")
+    set(Sanitizer_TYPE "ASAN" CACHE
```

Comment:
could you change the name to `SANITIZER_TYPE` (all uppercase)?

Reply:
Fixed the issue that it was forcing ASAN value and replaced Sanitizer_TYPE -> SANITIZER_TYPE